source/module_io/istate_envelope.cpp

```diff
@@ -4,7 +4,7 @@
 #include "module_base/global_variable.h"
 #include "module_io/write_wfc_pw.h"
 #include "module_io/write_wfc_r.h"
-#include "module_io/rho_io.h"
```

Comment:
I suggest not to remove rho_io.h, but rather add another cube_io.h, and functions in rho_io calls functions in cube_io

Reply:
In this way, the codes to create or open files will be able to be written inside write_rho and read_rho, respectively.

Reply:
OK

Reply:
It's done.